worldengine/astar.py

```diff
@@ -198,14 +197,6 @@ def _handle_node(self, x, y, from_node, destination_x, destination_y):
         return None
 
 
-def _matrix_to_array(matrix):
```

Comment:
This method ate huge amounts of time in comparison to everything else. I hope the replacement below is ok, but I am scratching my head about that a bit. (tests and output, however, are perfect)